proposals/geolocation.md

```diff
@@ -0,0 +1,404 @@
+# WoT Discovery
+## Geolocation
+Geolocation support for discovery includes two loosely-coupled
+specifications: an information model for geolocation data itself,
+which might be encoded in TDs or returned as data from properties,
+and a query mechanism to filter such data by location and time.
```

Comment:
do we want/need to distinguish between the term "position" and "location?

Reply:
probably I should be more careful in my terminlogy and use one or the other.  I'm not sure there is a distinction.  I do think is a little confusing to use "geolocation" for the link and then use "location" as just PART of the total set of geolocation information, however.  Something to discuss with the experts